README.md

```diff
@@ -1,2 +1,15 @@
-# emby-auto-organize
-Auto-organize plugin for Emby Server
+<h1 align="center">Jellyfin AutoOrganize Plugin</h1>
```

Comment:
I don't really like html in md, because a principle of md is that it should read well without the rendering. Just an opinion

Reply:
It's actually a copy of the README from the main repo